testsuite/sclang/CMakeLists.txt

```diff
@@ -3,5 +3,5 @@ file(GLOB tests *.scd)
 foreach(test ${tests})
     get_filename_component(test_name ${test} NAME_WE)
     set(test_target ${test_name}_run)
-    add_test(NAME ${test_target} COMMAND sclang -a -l ${CMAKE_BINARY_DIR}/build_sclang.cfg ${test})
+    add_test(NAME ${test_target} COMMAND sclang -l ${CMAKE_BINARY_DIR}/build_sclang.cfg ${test})
```

Comment:
I grepped for "build_sclang.cfg" and another place showed up:
https://github.com/supercollider/supercollider/blob/00476b2a955402d9beff198c65faee5654a154fb/CMakeLists.txt#L574
Any reason why you wouldn't do that there too?

Reply:
No. Will change here also.